src/drivers/pwm_out_sim/PWMSim.cpp

```diff
@@ -497,53 +210,17 @@ PWMSim::ioctl(device::file_t *filp, int cmd, unsigned long arg)
 
 	case PWM_SERVO_GET_COUNT:
 	case MIXERIOCGETOUTPUTCOUNT:
-		if (_mode == MODE_16PWM) {
-			*(unsigned *)arg = 16;
-
-		} else if (_mode == MODE_8PWM) {
-
-			*(unsigned *)arg = 8;
-		}
-
+		*(unsigned *)arg = OutputModuleInterface::MAX_ACTUATORS;
 		break;
 
 	case MIXERIOCRESET:
-		if (_mixers != nullptr) {
-			delete _mixers;
-			_mixers = nullptr;
-			_groups_required = 0;
-		}
-
+		_mixing_output.resetMixerThreadSafe();
 		break;
 
 	case MIXERIOCLOADBUF: {
 			const char *buf = (const char *)arg;
 			unsigned buflen = strnlen(buf, 1024);
```

Comment:
Where does this 1024 value come from?

Reply:
Let's get rid of it then: https://github.com/PX4/Firmware/pull/13205/commits/7bbcc150a6b38540f6cf568ffa329ba25b1f5244
It's not needed, the buffer is guaranteed to be null-terminated.